WordPress/Classes/ViewRelated/Post/PostCardTableViewCell.m

```diff
@@ -331,6 +331,10 @@ - (void)configureCardImage
     NSURL *url = [post featuredImageURLForDisplay];
     self.postCardImageView.image = nil; // Clear the image so we know its not stale.
     if ([post isPrivate] && [post.blog isHostedAtWPcom]) {
+        CGSize imageSize = self.postCardImageView.frame.size;
```

Comment:
There is no guarantee the cell has the correct frame when this method is called. For instance, if the cell was just dequeued and has not yet been through a layout pass its frame will still match what is in the xib. 
For the height it might be better to use `postCardImageViewHeightConstraint.constant` 
For the width maybe just use the `UIApplication.sharedApplication().keyWindow.frame.size.width` unless the current trait collection indicates its an iPad, and in that case use `maxIPadWidthConstraint.constant`.